docs/api/tutorials/lineage.md

```diff
@@ -131,3 +131,64 @@ You can now see the lineage between `fct_users_deleted` and `logging_events`.
 You can now see the column-level lineage between datasets. Note that you have to enable `Show Columns` to be able to see the column-level lineage.
 
 ![column-level-lineage-added](../../imgs/apis/tutorials/column-level-lineage-added.png)
+
+## Read Lineage
+
+<Tabs>
+<TabItem value="graphql" label="GraphQL" default>
+
+```json
+mutation searchAcrossLineage {
+  searchAcrossLineage(
+    input: {
+      query: "*"
+      urn: "urn:li:dataset:(urn:li:dataPlatform:dbt,long_tail_companions.adoption.human_profiles,PROD)"
+      start: 0
+      count: 10
+      direction: DOWNSTREAM
+      orFilters: [
+        {
+          and: [
+            {
+              condition: EQUAL
+              negated: false
+              field: "degree"
+              values: ["1", "2", "3+"]
+            }
+          ]
```

Comment:
can you add a comment here that additional search filters can be included here as well